app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,15 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(self, nickname: str, musical_instrument: str,
+                 bow_level: int) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._bow_level = bow_level
+
+    def player_info(self) -> str:
+        return (f"Elf ranger {self.nickname}. {self.nickname} "
+                + f"has bow of the {self._bow_level} level")
+
+    def get_rating(self) -> int:
+        return 3 * self._bow_level
```

Comment:
The `get_rating` method returns three times the `bow_level`, which is appropriate for this class. Ensure that this method aligns with the expected behavior in the context where it's used.